junifer/storage/base.py

```diff
@@ -103,7 +97,7 @@ def list_features(
         )
 
     @abstractmethod
-    def read_df(
+    def read_features(
```

Comment:
This is kind of tricky.

`read_features` should be able to read any kind of features: tables, matrix, nifti, etc.

The previous name `read_df`, aimed to tell you which kind of object you were getting.

I'm sill not sure about this change. We might need to discuss it in the channel.

Reply:
I see your point.

Reply:
yes, at it really depends on the storage.